Clarify salt master fingerprint instructions (#742)
SUSE/doc-susemanager: modules/administration/pages/master-fingerprint.adoc

See the following procedure:

> . On the master enter the following command as root and note the fingerprint labeled as ``master.pub``:

Reviewer: ```suggestion
. On the master, at the command prompt, as root, use this command to find the ``master.pub`` fingerprint:
```

Reviewer 2: @Loquacity lol you beat me to it, looks good!
